source/core/ut_utils.pkb

```diff
@@ -328,5 +328,33 @@ create or replace package body ut_utils is
     end if;
   end;
 
+  function to_xpath(a_list varchar2) return varchar2 is
+    l_xpath varchar2(32767) := a_list;
+  begin
+    if l_xpath not like '/%' then
+      l_xpath := '//'||replace(l_xpath,',','|//');
```

Comment:
My thinking was:
- Let users use CSV for simplicity, as it's very easy to type CSV list
- let them use list, as it gives more power and can be easily managed programmatically 
I don't think we should move away from CVS approach as it was already introduced in 3.0.0 for cursors.
I need to better understand the difference between CSV and XPath so we can distinguish one from another. That should be quite simple I think, but it's not yet implemented.